databroker/src/grpc/kuksa_val_v2/val.rs

```diff
@@ -646,9 +646,10 @@ impl proto::val_server::Val for broker::DataBroker {
                                             },
                                             Some(PublishValuesRequest(publish_values_request)) => {
                                                 let response = publish_values(&broker, &publish_values_request).await;
-                                                if let Err(err) = response_stream_sender.send(Ok(response)).await
-                                                {
-                                                    debug!("Failed to send response: {}", err);
+                                                if let Some(value) = response {
```

Comment:
We should better clarify in proto files when response shall be sent. Today my interpretation of the text below is that you ALWAYS should get a Response message. We should better clarify with something like "_Databroker returns PublishValuesResponse upon error, and nothing upon success_"

Current text:

 _Provider sends PublishValuesRequest -> Databroker returns PublishValuesResponse_

Similar for other similar cases